pkg/workloads/workload.go

```diff
@@ -0,0 +1,207 @@
+package workloads
+
+import (
+	"time"
+
+	"github.com/newrelic/newrelic-client-go/internal/serialization"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+)
+
+// Workload represents a New Relic One workload.
+type Workload struct {
+	Account             *AccountReference        `json:"account,omitempty"`
+	CreatedAt           *serialization.EpochTime `json:"created_at,omitempty"`
+	CreatedBy           *UserReference           `json:"created_by,omitempty"`
+	Entities            []*EntityRef             `json:"entities,omitempty"`
+	EntitySearchQueries []*EntitySearchQuery     `json:"entitySearchQueries,omitempty"`
+	EntitySearchQuery   string                   `json:"entitySearchQuery,omitempty"`
+	GUID                *string                  `json:"guid,omitempty"`
+	ID                  *int                     `json:"id,omitempty"`
+	Name                *string                  `json:"name,omitempty"`
+	Permalink           *string                  `json:"permalink,omitempty"`
+	ScopeAccounts       *ScopeAccounts           `json:"scopeAccounts,omitempty"`
+	UpdatedAt           *serialization.EpochTime `json:"updated_at,omitempty"`
+}
+
+// AccountReference represents the account this workload belongs to.
+type AccountReference struct {
+	ID   int    `json:"id,omitempty"`
+	Name string `json:"name,omitempty"`
+}
+
+// EntityRef represents an entity referenced by this workload.
+type EntityRef struct {
+	GUID string `json:"id,omitempty"`
+}
+
+// EntitySearchQuery represents an entity search used by this workload.
+type EntitySearchQuery struct {
+	CreatedAt *time.Time               `json:"createdAt,omitempty"`
```

Comment:
`*serialization.EpochTime` ?

Reply:
That was my first attempt, but it turns out it's not returned as an epoch.

Reply:
But UpdatedAt is?  We should raise that with the Workloads group if true

Reply:
I did and they are looking into it.

Reply:
👍